controllers/predicate/predicate_test.go

```diff
@@ -114,13 +115,99 @@ var _ = Describe("Druid Predicate", func() {
 		var pred predicate.Predicate
 
 		JustBeforeEach(func() {
-			pred = LeaseHolderIdentityChange()
+			pred = SnapshotLeaseChanged()
```

Comment:
Should be renamed to `SnapshotRevisionChanged`

Reply:
Done